http/http_api.go

```diff
@@ -0,0 +1,102 @@
+package http
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/gin-gonic/gin"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/haveachin/infrared"
+)
+
+var ConfigPath = "./configs"
+var ProxyGateway = infrared.Gateway{}
+
+// StartWebserver Start Webserver if environment variable "api-enable" is set to true
+func StartWebserver(configPath string, gateway infrared.Gateway) {
+	ProxyGateway = gateway
+	ConfigPath = configPath
+	if getEnv("API-ENABLED", "false") == "true" {
+		apiBind := getEnv("API-BIND", "127.0.0.1:8080")
+
+		fmt.Println("Starting WebAPI on " + apiBind)
+		server := gin.Default()
+		gin.SetMode(gin.ReleaseMode)
+
+		server.POST("/proxies/", addProxy)
+		server.DELETE("/proxies/:file/", removeProxy)
+
+		err := server.Run(apiBind)
+		if err != nil {
+			panic(err)
```

Comment:
panic might be a bit overkill here, use something like log.Fatal here or return an error instead